### PR TITLE
Enable latest TizenRT build

### PR DESCRIPTION
**Pull request by hs0225**
- Because TizenRT build method changed, `iotjs_main_tizenrt.c` is needed in the src folder.
 - Update doc
 - Update travis script

IoT.js-DCO-1.0-Signed-off-by: Hosung Kim hs852.kim@samsung.com

### CODE REVIEW

File: docs/build/Build-for-ARTIK053-TizenRT.md [hunk 1]
```diff
@@ -16,9 +16,19 @@ Untar the gcc-arm-none-eabi-4_9-2015q3-20150921-linux.tar and export the path li
...
+##### Install genromfs to use romfs
+
+```bash
+sudo apt-get install genromfs
 ```
 
-* Get IoT.js and TizenRT sources
+##### Install kconfig-frontends to use menuconfig
+
+Refet to [Kconfig-frontends Installation](https://github.com/Samsung/TizenRT#appendix)
```
Comment (daeyeon) [inline docs/build/Build-for-ARTIK053-TizenRT.md]: Typo: Refer

File: docs/build/Build-for-ARTIK053-TizenRT.md [hunk 2]
```diff
@@ -28,68 +38,56 @@ cd iotjs-tizenrt
...
-  |      + tizenrt
-  |          + artik05x
   + TizenRT
 ```
 
-#### 2. Add IoT.js as a builtin application for TizenRT
+#### 2. Configure IoT.js
+
+This step is optional. You can edit the build config of IoT.js in `iotjs/build.config`.
+For example, edit `iotjs-include-module` to add module as below.
```
Comment (daeyeon) [inline docs/build/Build-for-ARTIK053-TizenRT.md]: module**s**

File: docs/build/Build-for-ARTIK053-TizenRT.md [hunk 3]
```diff
@@ -28,68 +38,56 @@ cd iotjs-tizenrt
...
 
 ```bash
-cd ../../os
-make context
-cd ../../iotjs
-./tools/build.py --target-arch=arm --target-os=tizenrt --sysroot=../TizenRT/os --target-board=artik05x --clean
+make IOTJS_ROOT_DIR=../../iotjs
 ```
 
+If there is no `IOTJS_ROOT_DIR` argument, [IoT.js](https://github.com/Samsung/TizenRT/tree/master/external/iotjs) released in TizenRT is built, not `iotjs-tizenrt/iotjs`.
```
Comment (daeyeon) [inline docs/build/Build-for-ARTIK053-TizenRT.md]: My suggestion is like below:

If `IOTJS_ROOT_DIR` isn't given as the argument, IoT.JS codes, which is located in `external/iotjs` of TizenRT, will be used for the build.

File: tools/travis_script.py [hunk 4]
```diff
@@ -21,48 +21,76 @@
...
 
+def set_release_config_tizenrt():
+    exec_docker(DOCKER_ROOT_PATH, ['cp', 'tizenrt_release_config',
+                                   fs.join(DOCKER_TIZENRT_OS_PATH, '.config')])
+
 if __name__ == '__main__':
-    config = get_config()
+    # Get os dependency module list
+    target_os = os.environ['TARGET_OS']
     os_dependency_module = {}
```
Comment (daeyeon) [inline tools/travis_script.py]: delete this.

**Comment by hs0225** [comment]
@daeyeon I updated my PR.

**Comment by daeyeon** [comment]
https://github.com/Samsung/iotjs/issues/1229 is created for understanding.

**Pull request merged**